runtime/doc/builtin.txt

```diff
@@ -7896,6 +7901,16 @@ setcharsearch({dict})					*setcharsearch()*
 		Can also be used as a |method|: >
 			SavedSearch()->setcharsearch()
 
+setcmdline({str} [, {pos}])					*setcmdline()*
+		Set the command line to {str} and set the cursor position to
+		{pos}.
+		If {pos} is omitted, the last position is used.
+		Returns FALSE when successful, TRUE when not editing the
+		command line.
+
+		Can also be used as a |method|: >
+			SetCmdline()->setcmdline()
```

Comment:
How about a better name for the first function? For example:
```
GetText()->setcmdline()
```

Reply:
OK.